src/unix/linux_like/mod.rs

```diff
@@ -1480,6 +1490,9 @@ safe_f! {
 }
 
 extern "C" {
+    pub fn __libc_current_sigrtmax() -> ::c_int;
+    pub fn __libc_current_sigrtmin() -> ::c_int;
```

Comment:
I think it's fine to expose but I'd like to make it `#[doc(hidden)]`.

Reply:
Done. :+1: